CONTRIBUTING.md

```diff
@@ -261,8 +261,11 @@ Each PR should have one commit once it lands on `master`; this can be accomplish
    - Bump P2P and block protocol versions in  `version.go`, if necessary
```

Comment:
One line up, but do we still need to bump the Tendermint version in version.go? Where (if anywhere) do we need to specify the new version?

Reply:
The new version is picked up via `git describe`. No need to specify it anymore, but we still need to bump the other versions.